pkg/logs/input/docker/tailer.go

```diff
@@ -46,10 +46,11 @@ type Tailer struct {
 	shouldStop    bool
 	stop          chan struct{}
 	done          chan struct{}
+	unexpectedEOF chan string
```

Comment:
I think this channel should be renamed into something more friendly for its consumer, it's misleading as we don't know what it contains (a `containerID`).